libraries/EEPROM/src/utility/stm32_eeprom.c

```diff
@@ -100,6 +100,7 @@ extern "C" {
 #endif /* FLASH_BASE_ADDRESS */
 
 #if !defined(DATA_EEPROM_BASE)
+static bool is_buffer_filled = false, do_auto_flush = true;
```

Comment:
See my comment below about `is_buffer_filled` usage
```suggestion
static bool do_auto_flush = true;
```